### DISCUSSION TITLE
Will you add a pico4 controller emulation mode?

### DISCUSSION THREAD

**Discussion by Lyon-Jiang** [alvr-org/ALVR#1521] [General]
Now the controllers do not match reality.
@zarik5

**Comment by zmerp** [comment]
I don't have a Pico 4 for testing. @galister did you do some tests to find the optimal controller offset? In any case in a month i may be able to get my hands on a Pico 4, but i'm not certain.

**Comment by galister** [comment]
My experience is that the offsets seem to be different for people. What works for me is very far off for someone else and their offsets are very far off for me. It could be tied to hmd or controller firmware versions (or regions), but I did not look that far into it.
@Lyon-Jiang suggest trying to find the offsets that work for you in the meantime, and just stick to those.

**Reply by Lyon-Jiang** [reply]
My problem is just that the direciton of the controllers do not match reality, that makes the experience a bit uncomfortable, but I don't know which parameter to adjust.

**Reply by galister** [reply]
Yes, you can do this yourself by adjusting the controller offsets under the headset tab (click advanced on top right). It's the ones called Controller position offset and Controller rotation offset. For me the position was fine as it, and the rotation ended up being -20, 0, 0 (from top to bottom). You may need different values, though.

**Reply by Lyon-Jiang** [reply]
ths
